lib/couchdb/index.js

```diff
@@ -84,23 +85,56 @@ const upsert = R.curry((couch, bucket, keys, doc) =>
 )(doc)))
 
 
+const _findOneEqById = R.curry((couch, bucket, predicates) =>
+  getById(couch, bucket, predicates.id)
+  .then(R.when(Util.notNil, (doc) => R.compose(
+    R.ifElse(
+      R.equals(ID.couchify(predicates))
+    , R.always(doc)
+    , R.always(null)
+    )
+  , R.pick(R.keys(ID.couchify(predicates)))
```

Comment:
Should ID.couchify(predicates) be assigned to a variable since you use it twice....

```
couchfiedPredicates = ID.couchify(predicates)
return R.compose(
.....
```